core/src/impl/KokkosExp_IterateTileGPU.hpp

```diff
@@ -527,9 +366,7 @@ struct DeviceIterateTile<4, PolicyType, Functor, void> {
           }
         }
       }
-    }
-    // LR
-    else {
+    } else {
```

Comment:
Did you mean to change this?

Reply:
I was choosing one of the implementations over the others and I didn't pay too much attention to choosing the best comments. If you feel strongly about it, I can go again through the file.